Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +24,15 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (!IsArrayNullOrEmpty(array))
+            {
+                for (int i = 0; i < array.Length / 2; i++)
+                {
+                    int buffer = array[i];
```

Comment:
these 3 lines are repeated in sort as well. Could you use a private function for both? :)

Reply:
Yes, there could have been a SwapElements function or something like that that switched 2 elements at given indices. Also, for the buffer creation, I could have made another function which would have had the option to expand or shrink the buffer (allowing reusability inside the remove and insert functions).